source/common/access_log/access_log_impl.cc

```diff
@@ -212,11 +212,7 @@ bool ResponseFlagFilter::evaluate(const StreamInfo::StreamInfo& info, const Http
 GrpcStatusFilter::GrpcStatusFilter(
     const envoy::config::filter::accesslog::v2::GrpcStatusFilter& config) {
   for (int i = 0; i < config.statuses_size(); i++) {
-    absl::optional<Grpc::Status::GrpcStatus> grpc_status =
-        Grpc::Utility::nameToGrpcStatus(config.statuses(i));
-    // Like with ResponseFlagFilter, the grpc_status will be valid because of config validation.
-    ASSERT(grpc_status.has_value());
-    statuses_.insert(grpc_status.value());
+    statuses_.insert(cfgToGrpcStatus(config.statuses(i)));
```

Comment:
nit: s/cfgToGrpcStatus/protoToGrpcStatus to be a bit more clear and consistent with the rest of the codebase?

Reply:
Agreed. Will change.

Reply:
Addressed in 569a63a.